docs/changelog.md

```diff
@@ -8,8 +8,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added
 
+- [Documentation](guide/angular.md) for using web components with Angular's standalone bootstrapping
```

Comment:
You don't need the file endings for the links. I quickly checked it locally and it works with just the context.

Reply:
That's correct (see https://vitepress.dev/guide/markdown#internal-links), but since all the existing links use the `.md` ending, I would leave it as is.